tardis/montecarlo/tests/test_rpacket.c

```diff
@@ -0,0 +1,19 @@
+#include "../src/rpacket.h"
+#include <assert.h>
+#include <stdlib.h>
+#include <Python.h>
+
+void test_rpacket_get_nu(){
+	rpacket_t * rp = (rpacket_t *) malloc(sizeof(rpacket_t));
+	double value = 10.2;
+	rpacket_set_nu(rp, value);
+	if( value != rpacket_get_nu(rp) ){
+		PyErr_Format(PyExc_AssertionError, "value %g not %g", value, rpacket_get_nu(rp));
+	}
+}
```

Comment:
No it doesn't test it. Thats the problem. when we run that command, the `setup.py` stores them in the build directory in the following manner
```
build
  |
  scripts
  temp -> where all the .o files are compiled
  lib -> where all the .so files are compiled
```
Now the conftest.py tries to find `test_rpacket.o` in the `lib` directory and fails to find it. This is happening due to the astropy_helpers. We need the conftest to somehow detect the `test_rpacket.o` file. You can reproduce it by `python setup.py build` and see the build directory.